tools/config.schema.yaml

```diff
@@ -0,0 +1,855 @@
+$schema: "http://json-schema.org/draft-07/schema#"
+description: RAPIDS configuration schema
+required:
+  - DATABASE_GROUP
+  - TIMEZONE
+  - CREATE_PARTICIPANT_FILES
+  - PHONE_CALLS
+  - PHONE_KEYBOARD
+  - PHONE_LOCATIONS
+
+definitions:
+  PROVIDER:
+    type: object
+    required: [COMPUTE, SRC_FOLDER, SRC_LANGUAGE, FEATURES]
+    properties:
+      COMPUTE:
+        type: boolean
+      FEATURES:
+        type: [array, object]
+      SRC_FOLDER:
+        type: string
+      SRC_LANGUAGE:
+        type: string
+        enum: [python, r]
+
+  DORYAB_BLUETOOTH_FEATURE:
+    type: object
+    required: [DEVICES, SCANS_MOST_FREQUENT_DEVICE, SCANS_LEAST_FREQUENT_DEVICE]
+    properties:
+      DEVICES:
+        type: array
+        uniqueItems: True
+        items:
+          type: string
+          enum: ["countscans", "uniquedevices", "meanscans", "stdscans"]
+      SCANS_MOST_FREQUENT_DEVICE:
+        type: array
+        uniqueItems: True
+        items:
+          type: string
+          enum: ["withinsegments", "acrosssegments", "acrossdataset"]
+      SCANS_LEASE_FREQUENT_DEVICE:
+        type: array
+        uniqueItems: True
+        items:
+          type: string
+          enum: ["withinsegments", "acrosssegments", "acrossdataset"]
+  
+  APPLICATION_CATEGORIES_FEATURE:
+        type: object
+        required: [CATALOGUE_SOURCE, CATALOGUE_FILE, UPDATE_CATALOGUE_FILE, SCRAPE_MISSING_CATEGORIES]
+        properties:
+          CATALOGUE_SOURCE:
```

Comment:
this one has to be an enum FILE or GOOGLE

Reply:
this one is still missing

Reply:
Fixed